src/Compilers/Core/AnalyzerDriver/AnalyzerManager.cs

```diff
@@ -265,12 +265,6 @@ internal bool IsSupportedDiagnostic(DiagnosticAnalyzer analyzer, Diagnostic diag
 
             foreach (var diag in supportedDiagnostics)
             {
-                if (HasNotConfigurableTag(diag.CustomTags))
```

Comment:
The change makes sense. However this means that there is no way to disable an analyzer that contains a notconfigurable trigger diagnostic. I think that is a problem but I can't think of any good solutions to it. We probably need to think about that scenario some more.